src/audio_to_image_model/audio_to_iamge_standalone_predict.py

```diff
@@ -39,21 +39,36 @@ def predict(model, audio_segments, device):
     return np.array(predictions)
 
 
+# def generate_csv(predictions, threshold, row_id_prefix, label_to_bird_map):
+#    result_rows = []
```

Comment:
coudnt you just delete it?

Reply:
I am currently using both csv functions until I know which one to take for sure